scripts/templates/webdriverio.tpl.d.ts

```diff
@@ -49,6 +49,24 @@ declare namespace WebdriverIO {
         };
     }
 
+    interface CucumberOptions {
+        require?: string[],
+        backtrace?: boolean,
+        requireModule?: (string | object)[],
+        dryRun?: boolean,
+        failFast?: boolean,
+        format?: string[],
+        colors?: boolean,
+        snippets?: boolean,
+        source?: boolean,
+        profile?: string[],
+        strict?: boolean,
+        tags?: string[],
```

Comment:
Why do you removed tags from the template but have it here, can you make it consistent?

Reply:
Sorry, removed